extras_install.sh

```diff
@@ -48,13 +48,18 @@ if [ ${ID} = "mendel" ]; then
 fi
 
 if [ "${os_name}" = "Ubuntu" ] && [ "${os_version}" = "20.04" ]; then
-   python3 -m pip  install tflite_runtime==2.8.0
+   # For ubuntu 20.04 install tflite runtime == 2.8.0
+   python3 -m pip install tflite_runtime==2.8.0
 else
 
+   # For every other platform install tflite runtime by constraucting a url.
    py=$(python3 -V | awk '{print $2}' | awk -F. '{print $1 $2}')
+   # Get the python version for this platform
    arch=$(uname -m)
+   # Get the architecture for this platform.
    url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n $py; echo -n "-cp"; echo -n $py; echo -n "m-linux_"; echo -n ${arch}; echo -n ".whl")
```

Comment:
You can probably combine multiple trailing echo statement with something like:

url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n "${py}-cp${py}m-linux_${arch}.whl")